Create ui-c8y-1023-17-12-added-fix-for-data-point-graph-when-you-export-as-image.md (#4071)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-17-12-added-fix-for-data-point-graph-when-you-export-as-image.md

---
date: ""

> title: Added fix for data point graph when you export as image.

Reviewer: ```suggestion
title: Data point graph correctly displayed when exported as an image
```
